Formula/python@2.rb

```diff
@@ -87,15 +87,23 @@ def install
     ldflags  = []
     cppflags = []
 
+    # Need this to use homebrew installed libraries.
+    if HOMEBREW_PREFIX.to_s != "/usr/local"
+      cppflags << "-I#{HOMEBREW_PREFIX}/include"
+      ldflags << "-L#{HOMEBREW_PREFIX}/lib"
```

Comment:
@m000 is this for `gdbm` only or is this required for finding other dependencies too when homebrew is not in `/usr/local`?